src/common/PresenceStatusIndicator.js

```diff
@@ -67,11 +66,13 @@ function MaybeOpaqueBackgroundWrapper(
   |}>,
 ) {
   const { useOpaqueBackground, style, children } = props;
+  const context = useContext(ThemeContext);
```

Comment:
tiny nit:

I've more often skipped defining `context`, and done something like this instead:

```js
const { backgroundColor } = useContext(ThemeContext);
```

Or I guess if that conflicts with something, give it a different name like

```js
const { backgroundColor: themeBackgroundColor } = useContext(ThemeContext);
```

What do you think?

Reply:
Good call, done :)